src/website/app/pages/Color/color.md

```diff
@@ -0,0 +1,55 @@
+# Color
+
+Mineral UI color provides themes leading with inspiring hues and grounded in usable grays.
+Mineral UI is committed to providing an easy path to creating accessible palettes and themes.
+
+## Guidelines
+
+Color is used to establish focal points or highlight important information.
+Mineral UI uses color as a tool for communication and secondarily as decoration.
+Overuse of color will overload the viewer and undermine any effect it had in creating a visual hierarchy.
+
+Take care when overriding individual theme values, since they enable consistent styling through automatic application of tone and shadowing.
+
+## Themes
+
+A **theme** is composed of a hue color ramp and the base gray ramp. Every theme uses the base gray ramp.  
+
+You can create your own theme:
+  - `<value>_60` represents the base color for the theme.
+  - Hue ramps are generated from 10 points placed on a Bezier curve with 3 anchors.
+  - The top point represents `<value>_10`.
+  - The middle point represents `<value>_60`.
+  - The final point represents `<value>_100`.
+  - Each value is evenly distributed. Some manual adjustment might be necessary.
+  - Depending on the mood and tone for the theme, some curves are steeper and some more shallow. The theme will appear brighter and more saturated with shallower curves.
+
+![color swatch](/images/color-swatch.png)
+
+## Variants
+
+**Variants** are used in Mineral UI to aid users' understanding of the status of different callouts, actions, buttons etc.
+
+The color system features four variants to represent intention:
+- **Primary:** Actions only. The main action in the view.
+- **Success Variant:** Confirming Actions or Messages that were successful, or will begin a successful action.
+- **Warning Variant:** Actions or Messages that indicate a warning or may produce an undesirable result.
+- **Destructive Variant:** Actions or Messages that indicate errors or potential to destroy/remove data.
```

Comment:
I think the "Variant" in each of these is unnecessary. Also, the capitalization of "Actions" and "Messages" is odd.

Reply:
addressed